src/_temp_types/api/teams.ts

```diff
@@ -0,0 +1,28 @@
+export type ApiProjectSet = {
+    id: number
+    name: string
+    teams: ApiProject[]
+    createdAt: string
+    updatedAt: string
+}
+
+export type ApiProject = {
+    id: number
+    name: string
+    members: ApiCourseMember[] // This is supposed to be empty
```

Comment:
if it's supposed to be empty why is it here. There is no reason the frontend and backend types need to be 1:1.

Projects cannot have students. This is the logic the frontend should work on and that is the logic the types on the frontend should enforce